src/cljs/athens/devcards/styling_with_stylefy.cljs

```diff
@@ -1,136 +0,0 @@
-(ns athens.devcards.styling-with-stylefy
-  (:require
-    [athens.db]
-    [cljsjs.react]
-    [cljsjs.react.dom]
-    [devcards.core :refer-macros [defcard-doc]]))
```

Comment:
@shanberg Similar question that I had in style_guide.cljs. Though it's less relevant now that we can use styled-components, it took a while to figure out the intricacies of Stylefy, and I want that knowledge to be easily accessible in the future. Not sure we need it to be in storybook, but it should be somewhere. The code here could be a handy, evergreen reference.

Reply:
The only additions this documentation contains over the actual Stylefy documentation are the convention of appending `-style` to component style definition names and the preference for using the SCSS-like `stylefy/manual` mode rather than sub styles.

I'm not sure it’s worth keeping around if we extract Stylefy from the codebase, which I see as the natural final step in the transition.

Reply:
See https://github.com/athensresearch/athens/pull/1564#discussion_r704565717